config/clusters/nasa-cryo/common.values.yaml

```
@@ -63,6 +63,13 @@ basehub:
           scope:
             - read:org
     singleuser:
+      extraFiles:
+        jupyter_server_config.json:
```

A comment like this demystifies how this works quite a bit I think, and may help us avoid broken config due to changes in either basehub or here.

```suggestion
        # jupyter_server_config.json is defined by basehub, this entry adds to it
        jupyter_server_config.json:
```